src/gui/screen_menu_lan_settings.cpp

```diff
@@ -55,7 +55,7 @@ class MI_NET_IP_t : public WI_SWITCH_t<2> {
 public:
     MI_NET_IP_t()
         : WI_SWITCH_t(0, string_view_utf8::MakeCPUFLASH((const uint8_t *)label), 0, is_enabled_t::yes, is_hidden_t::no, string_view_utf8::MakeCPUFLASH((const uint8_t *)str_DHCP), string_view_utf8::MakeCPUFLASH((const uint8_t *)str_static)) {
-        this->index = netdev_get_ip_obtained_type() == NETDEV_STATIC
+        this->index = netdev_get_ip_obtained_type(netdev_get_active_id()) == NETDEV_DHCP
```

Comment:
Can this code be reached with networking turned off (set to `None`)? If so, this'll blow up.